internal/models/user.go

```diff
@@ -69,6 +61,21 @@ type User struct {
 	IsAnonymous bool       `json:"is_anonymous" db:"is_anonymous"`
 
 	DONTUSEINSTANCEID uuid.UUID `json:"-" db:"instance_id"`
+
+	OneTimeTokens []OneTimeToken `json:"-" has_many:"one_time_tokens"`
+
+	// Deprecated: use the one_time_tokens model instead
+	ConfirmationToken string `json:"-" db:"confirmation_token"`
+	// Deprecated: use the one_time_tokens model instead
+	RecoveryToken string `json:"-" db:"recovery_token"`
+	// Deprecated: use the one_time_tokens model instead
+	EmailChangeTokenCurrent string `json:"-" db:"email_change_token_current"`
+	// Deprecated: use the one_time_tokens model instead
+	EmailChangeTokenNew string `json:"-" db:"email_change_token_new"`
+	// Deprecated: use the one_time_tokens model instead
+	PhoneChangeToken string `json:"-" db:"phone_change_token"`
+	// Deprecated: use the one_time_tokens model instead
+	ReauthenticationToken string `json:"-" db:"reauthentication_token"`
```

Comment:
Maybe we should just remove them and see if there are build failures, that way we're sure it's no longer used. Commenting them out is fine also.